{
  try {
    redo();
  }
 catch (  CannotRedoException e) {
    System.out.println("Cannot redo: " + e);
  }
}
